vendor/gx/ipfs/QmPEpj17FDRpc7K1aArKZp3RsHtzRMKykeK9GVgn4WQGPR/go-ipfs-config/init.go

```diff
@@ -62,9 +62,9 @@ func Init(out io.Writer, nBitsForKeypair int) (*Config, error) {
 			Writable:     false,
 			PathPrefixes: []string{},
 			HTTPHeaders: map[string][]string{
-				"Access-Control-Allow-Origin":  []string{"*"},
-				"Access-Control-Allow-Methods": []string{"GET"},
-				"Access-Control-Allow-Headers": []string{"X-Requested-With", "Range"},
+				"Access-Control-Allow-Origin":  {"*"},
+				"Access-Control-Allow-Methods": {"GET"},
+				"Access-Control-Allow-Headers": {"X-Requested-With", "Range"},
```

Comment:
Are you required to change these vendor files to satisfy the linter? Would prefer that non-required edits are not included in these complicated PRs. FYI, in a future version of go, vendors will be checksum-ed and verified...

Reply:
Appreciate you keeping these changes within a single commit, though. 🤝

Reply:
Do you know the command to omit the vendor package from gofmt?

Reply:
I'm pretty sure `gofmt` should be aware of vendors and should not change other package's formatting.